dotcom-rendering/src/web/lib/verticalDividerWithBottomOffset.ts

```diff
@@ -16,7 +17,7 @@ export function verticalDividerWithBottomOffset(
 				width: 1px;
 				/* 100% is a reasonable fallback for browsers which don't support calc() */
 				height: 100%;
-				height: calc(100% - ${bottomPaddingSize});
+				height: calc(100% + ${bottomPaddingSize});
```

Comment:
This has flipped because the offset now has to grow into the `gap` rather than shrink away from the padding.